.gitignore

```diff
@@ -1,6 +1,6 @@
 .env
-.idea
-.vscode
+/.idea
+/.vscode
```

Comment:
What's the purpose behind this change? Either we want configs or we don't from IDEs... right?

Reply:
I've reverted those changes.

Generally the intention is to git-ignore all IDE specific config files.